dev/release/verify-release-candidate.sh

```diff
@@ -215,7 +215,11 @@ setup_tempdir() {
 setup_miniconda() {
   # Setup short-lived miniconda for Python and integration tests
   if [ "$(uname)" == "Darwin" ]; then
-    MINICONDA_URL=https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+    if [ "$(uname -m)" == "arm64" ]; then
+	MINICONDA_URL=https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh
+    else
+        MINICONDA_URL=https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
```

Comment:
Shouldn't we use miniforge in all cases?

Reply:
It doesn't really make a difference in the following steps, thus I would keep as-is.